api-svc/raid-api/src/main/java/au/org/raid/api/model/datacite/DataciteIdentifier.java

```diff
@@ -0,0 +1,23 @@
+package au.org.raid.api.model.datacite;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+@Component
+@Getter
+@Setter
+public class DataciteIdentifier {
+
+    private String identifier;
+    private String identifierType;
+
+    public DataciteIdentifier() {
```

Comment:
You can replace constructors by adding `@NoArgsConstructor` and `@AllArgsConstructor` above the class declaration.